src/obfuscation.ts

```diff
@@ -15,6 +15,38 @@ export function decodeBase64(input: string) {
   return base64.atobPolyfill(input);
 }
 
+export function obfuscatePrecomputedBandits(
+  salt: string,
+  bandits: Record<string, IPrecomputedBandit>,
+): Record<string, IPrecomputedBandit> {
+  const obfuscatedBandits: Record<string, IPrecomputedBandit> = {};
+
+  Object.entries(bandits).map((entry) => {
+    const [banditKey, banditResult] = entry;
```

Comment:
minor: the map key is variation value, not bandit key:
```suggestion
    const [variationValue, banditResult] = entry;
```

Reply:
done